src/layouts/Header/components/HelpDropdown/HelpDropdown.tsx

```diff
@@ -25,13 +27,17 @@ type Item = {
   children: string
 }
 
-const removeSentryForm = () => {
-  document.body.style.overflow = ''
-  document.querySelector('#help-dropdown-widget')?.remove()
-}
-
 function HelpDropdown() {
-  useEffect(() => removeSentryForm, [])
+  const { data: form } = useQuery({
+    queryKey: ['HelpDropdownForm'],
+    queryFn: () => SentryUserFeedback.createForm(),
+  })
```

Comment:
This was a clean way to handle the async createForm call. Alternative is some cursed combination of useState/Effect/Memo.

(also wanted to try this out after Nick explained to me that TS query could be used for any async state management, not just network requests 😜)

Reply:
Hrrm, suspense would currently be enabled here, may want to play around with toggling this incase it brings up any odd issues.